quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/DatabasePropertyMappers.java

```diff
@@ -37,29 +49,25 @@ public static PropertyMapper<?>[] getDatabasePropertyMappers() {
                         .paramLabel("jdbc-url")
                         .build(),
                 fromOption(DatabaseOptions.DB_URL_HOST)
-                        .to("kc.db-url-host")
                         .paramLabel("hostname")
                         .build(),
                 fromOption(DatabaseOptions.DB_URL_DATABASE)
-                        .to("kc.db-url-database")
                         .paramLabel("dbname")
                         .build(),
                 fromOption(DatabaseOptions.DB_URL_PORT)
-                        .to("kc.db-url-port")
                         .paramLabel("port")
                         .build(),
                 fromOption(DatabaseOptions.DB_URL_PROPERTIES)
-                        .to("kc.db-url-properties")
                         .paramLabel("properties")
                         .build(),
                 fromOption(DatabaseOptions.DB_USERNAME)
+                        .mapFrom(DatabaseOptions.DB, DatabasePropertyMappers::resolveUsername)
```

Comment:
We should be able to just remove the username / password overrides: https://github.com/keycloak/keycloak/issues/39154